skglm/datafits/single_task.py

```diff
@@ -336,10 +342,21 @@ def full_grad_sparse(
         for j in range(n_features):
             grad_j = 0.
             for i in range(X_indptr[j], X_indptr[j + 1]):
-                tmp = y[X_indices[i]] - Xw[X_indices[i]]
-                if np.abs(tmp) < self.delta:
-                    grad_j += - X_data[i] * tmp
+                residual = y[X_indices[i]] - Xw[X_indices[i]]
+                if np.abs(residual) < self.delta:
+                    grad_j += - X_data[i] * residual
                 else:
-                    grad_j += - X_data[i] * np.sign(tmp) * self.delta
+                    grad_j += - X_data[i] * np.sign(residual) * self.delta
             grad[j] = grad_j / n_samples
         return grad
+
+    def intercept_update_step(self, y, Xw):
```

Comment:
Same here as logreg

Reply:
this is in Huber datafit so it's currently already used (beware of the diff that gets truncated, this class is not SVC)